app/uk/gov/hmrc/plasticpackagingtax/registration/views/partner/partner_registration_available_soon_page.scala.html

```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.models.Title
+@import uk.gov.hmrc.plasticpackagingtax.registration.config.AppConfig
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.request.AuthenticatedRequest
+
+@this(
+    govukLayout: main_template,
+    pageHeading: pageHeading,
+    paragraphBody: paragraphBody,
+    paragraph: paragraph,
+    govUkLink: govUkLink,
+    govukButton: GovukButton,
+    sectionHeader: sectionHeader,
+    link: link,
+    appConfig: AppConfig
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@isAuthenticated = @{request.isInstanceOf[AuthenticatedRequest[_]]}
+
+@feedbackUrl = @{
+    if (isAuthenticated) {
+        s"${appConfig.authenticatedFeedbackUrl()}&backUrl=${appConfig.selfBaseUrl}${request.uri}"
+    } else {
+        s"${appConfig.unauthenticatedFeedbackUrl()}&backUrl=${appConfig.selfBaseUrl}${request.uri}"
+    }
+}
+
+@linkFeedback = {<a href="@feedbackUrl" class="govuk-link" id="govuk-feedback-link">@messages("partnership.registrationAvailableSoon.feedback.link")</a>}
+
```

Comment:
Not a requirement of your ticket but it might be nice to extract this feedback link into a component (or page fragment). I've definitely seen it used in at least 1 other place. Perhaps one for the future.

Reply:
Maybe a low priority ticket on the backlog?